operator/pkg/validate/common.go

```diff
@@ -185,6 +186,11 @@ func validateCIDR(path util.Path, val any) util.Errors {
 	var err error
 	if !util.IsString(val) {
 		err = fmt.Errorf("validateCIDR %s got %T, want string", path, val)
+		// check the val whether to start and end with a space
+	} else if len(val.(string)) == 0 {
+		err = fmt.Errorf("validateCIDR %s got empty string, want not empty string", path)
+	} else if unicode.IsSpace(rune(val.(string)[0])) || unicode.IsSpace(rune(val.(string)[len(val.(string))-1])) {
+		err = fmt.Errorf("validateCIDR %s got space begin or end, want not begin with space or end", val)
```

Comment:
why do we need this when we already changed L115?

Reply:
Thanks to review, It’s not necessary here. I didn’t find that netip.ParsePrefix has been able to return a space error before.